examples/fizzbuzz/main.qml

```diff
@@ -0,0 +1,37 @@
+import QtQuick 2.2
+import QtQuick.Controls 1.1
+import QtQuick.Layouts 1.1
+import Examples.FizzBuzz 0.1
+
+ApplicationWindow {
+    visible: true
+    width: 200
+    height: 200
+    title: "FizzBuzz"
+
+    ColumnLayout {
+        anchors.fill: parent
+        anchors.margins: 10
+        TextField {
+            placeholderText: "Input"
+            text: "0"
```

Comment:
When "" is ignored, we can use that as initial text to not have a fizzbuzz from the very beginning.